torch/ao/quantization/_pt2e/_propagate_annotation.py

```diff
@@ -0,0 +1,45 @@
+import torch
+from torch.fx import Node
+from typing import (
+    List,
+    Callable,
+)
+
+def _is_share_obs_or_fq_op(op: Callable) -> bool:
+    return op in [
+        torch.ops.aten.hardtanh.default,
+        torch.ops.aten.mean.default,
+        torch.ops.aten.mean.dim,
+        torch.ops.aten.adaptive_avg_pool2d.default,
+        torch.ops.aten.view_copy.default,
+        torch.ops.aten.view.default,
+    ]
+
+def propagate_annotation(model: torch.fx.GraphModule) -> None:
+    for n in model.graph.nodes:
+        if n.op != "call_function" or not _is_share_obs_or_fq_op(n.target):
+            continue
+
+        prev_node = n.args[0]
+        if not isinstance(prev_node, Node):
+            continue
+
+        target_dtype_info = prev_node.meta.get("target_dtype_info", None)
+        if not target_dtype_info:
+            continue
+
+        output_act_obs_or_fq_ctr = target_dtype_info.get("output_act_obs_or_fq_ctr", None)
+        if not output_act_obs_or_fq_ctr:
+            continue
+
+        # make sure current node is not annotated
+        if "target_dtype_info" in n.meta and n.meta["target_dtype_info"].get("_annotated", False):
```

Comment:
From current implementation, looks like these ops in `_is_share_obs_or_fq_op` is still configurable by user. It will skip the `propagate_annotation` if it has already be annotated in `quantizer`. Should we add some check to prevent these ops accidently annotated by `quantizer`?

Reply:
I think that's allowed, it's fine if user wants to add some annotation for these ops, in that case we just don't do propagation. what I meant by not configurable is that the op_list is now fixed, user can't add new ops to this list right now

Reply:
Got it. Make sense and thanks.